CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 2.17.1 (2019-07-19)
+
+- Added logic to correctly format the zone when added to a tokenized URL. [#102](https://github.com/blackbaud/auth-client/pull/102)
```

Comment:
I'd add "it is" after "when" ... Also, I'd add "(Thanks @Blackbaud-JonathanBell!)" since we usually use that pattern for contributions. (I'd make the changes myself, but I don't seem to have rights to this repo. Sorry.)